.github/workflows/linting.yml

```diff
@@ -0,0 +1,26 @@
+name: Linting
+
+on:
+  pull_request:
+    branches:
+      - main
+
```

Comment:
Do we want the linting just on pull requests or any push to any branch (to alert the developers about linting issues early on)?

Reply:
@kate-holdener My thought behind this was, to reduce the number of times the action runs and also because we have one week after draft PR is raised. But I understand your point, will change it to all pushes.

Reply:
@kate-holdener, I have fixed it now. Also, I feel like we can merge this PR and I can create another PR for JS linting.